src/components/drs/pages/DrsShow.js

```diff
@@ -0,0 +1,231 @@
+import '@fontsource/roboto';
+import React, {
+  useState, 
+  useEffect
+} from 'react';
+import axios from 'axios';
+import { 
+  Typography, 
+  Container, 
+  Table,
+  TableContainer, 
+  TableHead, 
+  TableBody, 
+  TableRow, 
+  TableCell, 
+  List, 
+  ListItem, 
+  ListItemText
+} from '@material-ui/core';
+import {
+    useParams, 
+    Link
+} from "react-router-dom";
+
+const ChildProperties = (props) => {
+  const childPropertyKey = props.childPropertiesArray[0];
+  const childPropertyValueArray = props.childPropertiesArray[1];
+
+  if(childPropertyKey === 'drs_object_children' || childPropertyKey === 'drs_object_parents'){
```

Comment:
there's too much `if-else` logic here that's hard to parse out. It would go away if a `DrsObject` react component with awareness of `DrsObject` properties was employed